.github/workflows/windows_release.yml

```diff
@@ -30,8 +30,10 @@ jobs:
     - name: Install Chocolately Packages
       run: choco install miktex && refreshenv
 
-    - name: Add MikTeX binary folder to path
-      run: echo "::add-path::/C/Program Files/MikTeX/miktex/bin/x64"
+    - name: Build Test Document (will install missing packages)
```

Comment:
This is where I am trying to build a test document ahead of building the full set of docs in order to install the missing miktex packages cleanly in a single thread.